apps/prairielearn/elements/pl-multiple-choice/pl-multiple-choice.mustache

```diff
@@ -59,27 +59,80 @@
 {{/radio}}
 {{^radio}}
 <!-- Weirdly enough putting this in a div causes it to display without the inline -->
-<span class="py-1 d-inline align-items-center">
+<span
+    class="py-1 d-inline align-items-center pl-multiple-choice-dropdown"
+    {{#size}}style="--pl-multiple-choice-dropdown-width: {{size}}px"{{/size}}
+>
     <script>
```

Comment:
This script tag is now at ~60 lines with nothing other than the uuid as dynamic data. Maybe in a future PR, but i think it's worth moving this to a `pl-multiple-choice.js` file instead.

Reply:
I'll do it now 👍

Reply:
Done in 86983ba.